wercker.yml

```diff
@@ -0,0 +1,48 @@
+box: node
+
+build:
+  steps:
+    - script:
+        name: echo nodejs information
+        code: |
+          echo "node version $(node -v) running"
+          echo "npm version $(npm -v) running"
+
+build-latest:
+  steps:
+    - script:
+        name: echo nodejs information
+        code: |
+          echo "node version $(node -v) running"
+          echo "npm version $(npm -v) running"
+    - npm-install
+    - npm-test
```

Comment:
How much is this `build-latest` different from the `build` above?

Reply:
the default workflow in wercker must start with `build` pipeline. 
So for now the `build` do almost nothing - just printing the version - so the tests for all versions can be run in parallel as the next step after `build`.

However for now wercker doesn't allow fan-in for the pipelines so `release` pipeline cannot wait for the parallel tests.
So it is more prepared for the future.

Reply:
@tomas-mechura did you checked this with Wercker? I remember Ladislav mentioned they might have it available from yesterday.

Reply:
@freaz they added it 1 hour ago :)
updated the wercker workflow:
https://app.wercker.com/APIPCS/blueprint-markdown-renderer/workflows

Reply:
:D